src/Apps/Artist/Components/ArtistHeader.tsx

```diff
@@ -44,6 +43,12 @@ interface Props {
   mediator?: Mediator
 }
 
+const Categories = {
```

Comment:
This casing seems odd for a constant-- would probably make it `CATEGORIES`.

Also, below you are adding "Representation" to the string. Why not just include that here? (i.e. `"Blue Chip Representation"`)